commcare_export/env.py

```diff
@@ -217,6 +219,36 @@ def emitted_tables(self):
 # Actual concrete environments, basically with built-in functions.
 #
 
+@unwrap
+def str2bool(val):
+    if isinstance(val, bool):
+        return val
+    return val and val.lower() in {'true', 't', '1'}
```

Comment:
Looks like this function may be called with non-string arguments, otherwise `isinstance` call above would be unnecessary. Is it safe to assume `val` will always be either a string or bool? If not, should this to use `str(val).lower()` (or something more fancy if `val` could contain non-ascii Unicode characters)?